lib/splitclient-rb/split_config.rb

```diff
@@ -29,7 +29,7 @@ def initialize(opts = {})
       @events_uri = (opts[:events_uri] || SplitConfig.default_events_uri).chomp('/')
       @mode = opts[:mode] || SplitConfig.default_mode
       @redis_url = opts[:redis_url] || SplitConfig.default_redis_url
-      @redis_namespace = opts[:redis_namespace] || SplitConfig.default_redis_namespace
+      @redis_namespace = opts[:redis_namespace] ? "#{opts[:redis_namespace]}/#{SplitConfig.default_redis_namespace}" : SplitConfig.default_redis_namespace
```

Comment:
@patricioe @sarrubia 
There is a little ambiguity in the [docs](https://docs.google.com/document/d/10NSU3Fgmfa74ElM2mZGTqdNjwPG5mHmd5XSJG8X3nkA/edit), both `/` and `.` is mentioned there. What should I use?